solutions/과자.js

```diff
@@ -0,0 +1,34 @@
+const INPUT_PATH = "../inputs/과자.txt";
+
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : INPUT_PATH;
+const input = fs.readFileSync(filePath).toString().trim().split("\n");
+
+function solution(input) {
+  const [M, N] = input[0].split(" ").map(Number);
```

Comment:
`M`, `N`대신에 조금더 명시적인 변수이름을 설정해주면 좋을 것 같습니다!

ex)
```suggestion
  const [nephews, cookieLength] = input[0].split(" ").map(Number);
```